src/test/java/uk/gov/hmcts/reform/bulkscanprocessor/model/mapper/EnvelopeMapperTest.java

```diff
@@ -79,10 +84,27 @@ private void assertSameScannableItems(Envelope dbEnvelope, InputEnvelope zipEnve
 
         assertThat(dbEnvelope.getScannableItems())
             .extracting(this::convertToInputScannableItem)
-            .usingRecursiveFieldByFieldElementComparator()
+            .usingElementComparatorIgnoringFields("ocrData", "documentType")
             .containsAll(zipEnvelope.scannableItems);
     }
 
+    private void assertSameOcrData(Envelope dbEnvelope, InputEnvelope zipEnvelope) {
```

Comment:
I meant another test to keep assertions less per test and not affect existing test case,

Reply:
I agree - one test for this general verification and a separate test just focusing on those document types (there could be an envelope with all of them and the test would verify if each of them has been properly mapped, ignoring all other fields from the envelope).